2/ex2.py

```diff
@@ -0,0 +1,10 @@
+# A comment, this is so you can read your program later.
+# Anything after the # is ignored by python.
+
+print "I could have code like this." # and the comment after is ignored
```

Comment:
Awesome, glad you got to see that comments/code can live in conjunction like this. Two (quick) exercises for you (no need to commit anything):

1.Try writing and running something like this in a separate file:
```python
print '5'
"""Example comment."""
```

Is this valid? What does that second line do? It seems just like a comment! What is this useful for? The name for this is a `docstring` and you can actually write comments on multiple lines, without prefacing all of them with `#`. It's very useful for writing multi-line comments.
<br />2. What do you think comments are useful for? The obvious reason is to explain bits of your code that are confusing but do any other reasons come to mind?